server/graphql/v1/mutations/orders.js

```diff
@@ -28,34 +28,58 @@ const ordersLimitCache = LRU({
 const debugOrder = debug('order');
 
 function checkOrdersLimit(order) {
-  const fromCollectiveIdOrUserEmailKey = get(order, 'fromCollective.id') || get(order, 'user.email');
+  const fromCollectiveIdOrUserEmailKey =
+    get(order, 'fromCollective.id') || get(order, 'user.email');
   const collectiveIdKey = get(order, 'collective.id');
   // First of all, check if creation of orders has reached a limit but disabling behaviour
   // temporarily for "test" or "circleci" ENVs(both test environments) as most tests create more than 2 orders per (fromcollective,collective) and it fails
-  if (process.env.NODE_ENV !== 'test' && process.env.NODE_ENV !== 'circleci' &&
-      fromCollectiveIdOrUserEmailKey && collectiveIdKey) {
-    const fromCollectiveValue = ordersLimitCache.get(fromCollectiveIdOrUserEmailKey);
+  if (
+    process.env.NODE_ENV !== 'test' &&
+    process.env.NODE_ENV !== 'circleci' &&
+    fromCollectiveIdOrUserEmailKey &&
+    collectiveIdKey
+  ) {
```

Comment:
out of curiosity: are we following the up to 120 chars in one line, right? I personally prefer having less lines and filling out those lines with something closer to the 120 chars limi